src/styles/utils/borderStyle.ts

```diff
@@ -64,6 +70,58 @@ const borderRadius = css<{ $borderRadius?: BorderRadiusProps }>`
     parseBorderRadius,
   )}
 `;
+const borderRadiusTopLeft = css<BorderStyleProps>`
```

Comment:
I wanted to be able to control the border-radius for each corner in the tooltip. I could have added this with styled-components but I felt that this possibly belonged here.